src/main/java/Duke.java

```diff
@@ -1,10 +1,106 @@
+import java.util.Scanner;
 public class Duke {
+    public static Task[] list = new Task[100];
```

Comment:
For a collection of objects, you might want to consider using the plural form. So here instead of calling the array "list" you could call it "tasks" since it is a collection of tasks.